common/lib/xmodule/xmodule/vertical_block.py

```diff
@@ -81,7 +81,7 @@ def student_view(self, context):
                 child_block_context['wrap_xblock_data'] = {
                     'mark-completed-on-view-after-delay': complete_on_view_delay
                 }
-            rendered_child = child.render(STUDENT_VIEW, child_block_context)
+            rendered_child = child.render(context.get('view_type', STUDENT_VIEW), child_block_context)
```

Comment:
Rename this method `_student_view(self, context, view)` and make the two view methods pass `STUDENT_VIEW` or  `ANONYMOUS_VIEW` to it.

Reply:
👍